metadata-app/src/api/index.js

```diff
@@ -307,9 +307,9 @@ const fetchParentDatasets = async (query, apiUrl, apiKey) => {
         'X-CKAN-API-Key': apiKey,
       },
     });
-    return (res.data.result.results || []).map(({ id, name, title }) => {
+    return (res.data.result.results || []).map(({ id, title }) => {
       return {
-        id: name || id,
+        id,
```

Comment:
According to docs we could use name as id of a package but maybe something is different in custom extension for inventory app.

https://docs.ckan.org/en/2.5/api/index.html#ckan.logic.action.get.package_show